src/components/StageBar.vue

```diff
@@ -38,31 +38,51 @@ export default {
 @import '@/styles/theme.scss';
 
 .stage-bar {
-  display: flex;
-  flex-flow: row wrap;
-  justify-content: center;
-  padding-left: 16px;
-
+  position: relative;
+  display: grid;
+  grid-template-columns: auto auto auto;
+  grid-template-rows: 50px;
+  overflow: hidden;
   .link-btn {
-    color: $alt-grey-2;
-    font-weight: bold;
-    font-family: 'Roboto';
-    font-size: 16px;
-    padding: 24px;
-    text-decoration: none;
-    border: none;
-    border-bottom: 1.5px solid $alt-grey-1;
-    outline: none;
+    height: 50px;
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background: $purple-2;
+    color: $white;
     cursor: pointer;
-    background: none;
-
     .link-btn:active,
     &:focus,
     &.active {
-      outline: none;
-      border-bottom: 2px solid $purple-4;
-      color: $purple-4;
+      background: $purple-3;
+      color: $white;
     }
+    &.active::after {
+      border-left-color: $purple-3;
+    }
+  }
+  .link-btn::before,
+  .link-btn::after {
+    content: '';
+    position: absolute;
+    z-index: 2;
+    border-top: 31px solid transparent;
+    border-bottom: 31px solid transparent;
+    border-left: 12px solid;
+    right: -12px;
+    border-left-color: $alpha-purple;
+  }
+  .link-btn::after {
```

Comment:
The arrow should be bigger:
![image](https://user-images.githubusercontent.com/24905681/81174711-9c6bb600-8fa2-11ea-8c0a-02c08dbe5853.png)

![image](https://user-images.githubusercontent.com/24905681/81174675-8cec6d00-8fa2-11ea-923e-fc881dd6674a.png)